helpers/setup.py

```diff
@@ -176,9 +178,15 @@ def update_hosts(cls, dict_):
             config = Config()
             config.write_config()
 
-            cmd = 'sudo mv /etc/hosts /etc/hosts.old ' \
-                  '&& sudo mv /tmp/etchosts /etc/hosts'
+            cmd = (
+                'sudo cp /etc/hosts /etc/hosts.old '
+                '&& sudo cp {tmp_file_path} /etc/hosts'
+            ).format(tmp_file_path=tmp_file_path)
```

Comment:
Maybe rather this?

```python
cmd = (
    'sudo cp /etc/hosts /etc/hosts.old '
    '&& sudo cp {tmp_file_path} /etc/hosts'
).format(tmp_file_path)
```

or this

```python
cmd = (
    'sudo cp /etc/hosts /etc/hosts.old '
    '&& sudo cp {} /etc/hosts'
).format(tmp_file_path)
```

Reply:
Actually, I did it in purpose. 
Because I'm keeping in mind the drop support of Python3.5 and the beginning of using f-strings ;-) 
I'll keep it as-is.